plexpy/pmsconnect.py

```diff
@@ -1921,6 +1925,14 @@ def get_session_each(self, session=None):
 
         # Get the quality profile
         if media_type in ('movie', 'episode', 'clip') and 'stream_bitrate' in stream_details:
+            # Set the full resolution by combining video_resolution and video_scan_type
```

Comment:
`video_full_resolution` and `stream_video_full_resolution` should be outside the if statement.